ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -278,12 +333,20 @@ class AlertsView extends React.Component {
             </Alert>
           )}
           <AlertsViewControls
-            dropdownOptions={id ? [] : alertDropdowns}
+            isListMode={this.isListMode()}
```

Comment:
Suggestion: all filters can be grouped in a single object.
Benefit: less verbosity.